protocol/app/app.go

```diff
@@ -588,10 +588,22 @@ func New(
 	transferModule := transfer.NewAppModule(app.TransferKeeper)
 	transferIBCModule := transfer.NewIBCModule(app.TransferKeeper)
 
+	app.BlockTimeKeeper = *blocktimemodulekeeper.NewKeeper(
```

Comment:
This code block is moved from below to before the `app.RateLimitKeeper` initialization.